client/landing/subscriptions/components/add-sites-modal/styles.scss

```diff
@@ -12,5 +12,35 @@
 		font-style: normal;
 		font-weight: 400;
 		line-height: 20px;
+		margin-bottom: 1em;
+	}
+
+	.reader-feed-item__site-preview-h-stack {
+		max-width: 200px;
+	}
+
+	.reader-feed-item__description {
+		display: -webkit-box;
+		-webkit-line-clamp: 2;
+		-webkit-box-orient: vertical;
+		white-space: normal;
+	}
+
+	.feed-preview {
+		max-width: 586px;
+	}
+
+	.empty-content__title {
+		display: none;
+	}
+
+	.empty-content__line {
+		margin-bottom: 20px;
+	}
+
+	.list-end .gridicon {
```

Comment:
The WordPress logo at the end of the stream was looking to small with alignment disturbed so this fixes it.